javascript-bank/account.js

```diff
@@ -0,0 +1,43 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if ((amount <= 0) || (typeof amount !== 'number') || (isNaN(amount)) || (!(Number.isInteger(amount)))) {
+    return false;
+  } else if (amount > 0) {
```

Comment:
This work doesn't need to be an `else if`. Since you are `return`ing `false` in the above block, all of this work can just be below that conditional. This is the pattern we've discussed previously called a "guard". Once you've checked for all the possible errors and you `return` if there are any problems, you can do your regular work below that conditional safely.